src/common/libpmi/Makefile.am

```diff
@@ -9,7 +9,8 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir) \
 	-I$(top_srcdir)/src/include \
 	-I$(top_srcdir)/src/common/libccan \
-	-I$(top_builddir)/src/common/libflux
+	-I$(top_builddir)/src/common/libflux \
+	$(JANSSON_CFLAGS)
```

Comment:
Commit message nit: A Makefile doesn't check for flags, but instead specifies them for compilation and linking.

Perhaps something like 

> Problem: JANSSON_CFLAGS is not used in libpmi/Makefile.am, but the PMI code includes jansson.h.
> 
> Add JANSSON_CFLAGS to AM_CPPFLAGS so that jansson.h installed outside of the default include search path can be found.